src/screens/Lessons/screens/New/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import {newLessonsTitleText} from 'utils/text'
+import Split from 'components/Split'
+import Requested from './components/Requested'
+import Propose from './components/Propose'
+
+export default ({instructor}) => (
+  <Split
+    title={newLessonsTitleText}
+    main={
+      <Requested instructor={instructor} />
+    }
+    aside={
+      <Propose instructor={instructor} />
+    }
+  />
+)
```

Comment:
This view:
![image](https://cloud.githubusercontent.com/assets/5497885/22714673/d653b4ce-ed4a-11e6-8a46-8f7586e62082.png)